sdk/client/user_client.go

```diff
@@ -2,14 +2,15 @@ package client
 
 import (
 	"context"
-	"github.com/conductor-sdk/conductor-go/sdk/model/rbac"
 	"net/http"
+
+	"github.com/conductor-sdk/conductor-go/sdk/model/rbac"
 )
 
 type UserClient interface {
 	CheckPermissions(ctx context.Context, userId string, type_ string, id string) (map[string]interface{}, *http.Response, error)
 	DeleteUser(ctx context.Context, id string) (*http.Response, error)
-	GetGrantedPermissions(ctx context.Context, userId string) ([]rbac.GrantedAccess, *http.Response, error)
+	GetGrantedPermissions(ctx context.Context, userId string) (rbac.GrantedAccessResponse, *http.Response, error)
```

Comment:
The return type was wrong. Because of buggy error handling this was not caught by the tests before